CMakePresets.json

```diff
@@ -37,12 +40,12 @@
       "hidden": true
     },
     {
-      "name": "x64-release",
+      "name": "windows-tiles-sounds-x64-msvc",
```

Comment:
Need to add this to `buildPresets` at the bottom.

```
$ cmake --list-presets
Available configure presets:

  "windows-tiles-sounds-x64-msvc" - x64 Release
  "x64-debug"                     - x64 Debug

akrieger@SCOTTY MINGW64 /c/src/Cataclysm-DDA
$ cmake --build --preset windows-tiles-sounds-x64-msvc
CMake Error: No such build preset in C:/src/Cataclysm-DDA: "windows-tiles-sounds-x64-msvc"
Available build presets:

  "x64-debug"
```

Reply:
Removed `buildPresets` for now. It is in auto-generated `CMakeUserPresets.json`.